packages/playwright-test/src/testType.ts

```diff
@@ -57,6 +55,8 @@ export class TestTypeImpl {
     test.step = wrapFunctionWithLocation(this._step.bind(this));
     test.use = wrapFunctionWithLocation(this._use.bind(this));
     test.extend = wrapFunctionWithLocation(this._extend.bind(this));
+    test.projectSetup = wrapFunctionWithLocation(this._createTest.bind(this, 'projectSetup'));
+    (test.projectSetup as any).only = wrapFunctionWithLocation(this._createTest.bind(this, 'only'));
```

Comment:
This should be `projectSetupOnly`, otherwise it creates just a test.

Reply:
Good catch!